test/e2e/framework/test_context.go

```diff
@@ -277,6 +277,8 @@ type NodeTestContextType struct {
 	ExtraEnvs map[string]string
 	// StandaloneMode indicates whether the test is running kubelet in a standalone mode.
 	StandaloneMode bool
+	// CriProxyEnabled indicates whether enable CRI API proxy for failure injection.
```

Comment:
Someday this `NodeTestContextType` should get moved out of `test/e2e/framework` - but today is not that day, so:

/approve